labcodes/labstorybook
# DSYS-22/tags-qa

**Link to task:**
https://labcodes.atlassian.net/browse/DSYS-22 (QA 15/10)

**How to test:**
- With the LabSystem running, go to the "Component/Tag" div in the LabStoryBook, then check if everything is rendered as expected.
- Locally, from labstorybook dir, run `npm test`, then check if all tests are passing.

**Description of your solution:**
This PR fixes the documentation of Tags according to QA from 15/10/20, including "stories" and "playground". The architecture of 'TagPlayground' is refactored into a single dynamic component. Snapshots are udpated.
